src/qtiCommonRenderer/tpl/interactions/choiceInteraction.tpl

```diff
@@ -7,6 +7,7 @@
 >
   {{#if prompt}}{{{ prompt }}}{{/if}}
   <div class="instruction-container"></div>
+  <div class="header-dummy">Header</div>
```

Comment:
**question:** Why is it required? It looks a little dumb.

Reply:
at first look it may but, yes it's needed.

This is a placeholder is expanding the height of the header of the table (ex. when eliminated is on) 
the title itself is a part of the checkbox, cause as I understood it made to have an alignment relative to the position of the checkboxes (it varies).
when you change the font size, so this can eliminate overlap title (eliminate) with table and question section
before:
<img width="852" alt="Screenshot 2020-10-29 at 12 52 26" src="https://user-images.githubusercontent.com/2067027/97565214-23479b80-19e6-11eb-9bd8-bcf12238d8a6.png">

after:
<img width="1093" alt="Screenshot 2020-10-29 at 12 53 21" src="https://user-images.githubusercontent.com/2067027/97565238-2d699a00-19e6-11eb-91bb-11bc56224f77.png">